package.json

```diff
@@ -28,6 +28,7 @@
     "react-dom": "^18.2.0",
     "react-icons": "^4.8.0",
     "react-query": "^3.39.3",
+    "tbe-webapp": "file:",
```

Comment:
this change wasn't needed in this PR. Please remove this line

Reply:
Removed the unnecessary line as it wasn't required for this PR.